package.json

```diff
@@ -139,6 +139,9 @@
   "peerDependenciesMeta": {
     "webpack-cli": {
       "optional": true
+    },
+    "webpack": {
+      "optional": true
     }
```

Comment:
You may also want to remove it from the `peerDependencies`field  above

https://github.com/webpack/webpack-dev-server/blob/4e846078b3b9befafbc56b1cd6db7778175355c9/package.json#L137

Reply:
if we remove `webpack` from `peerDependencies` it would mean any `webpack` version would be ok for `webpack-dev-server` but `webpack-dev-server` did require specific `webpack` version according to the documentation of https://pnpm.io/next/package_json#peerdependenciesmetaoptional

Reply:
Yeah, we still need versions, in theory this PR can break a code for some inattentive developers (which ignored warnings from package managers), if you don't have webpack in dependcies and rely on the new behaviour of npm (autoinstall peer deps), but it should be fixed on their side, because you should always list of you dependecies